index.html

```diff
@@ -73,7 +86,10 @@ <h3 class="s3h3">Multi-Post Stories</h3>
           <button class="btn1">See Project</button>
```

Comment:
- [ ] Any of your `See Project` buttons are not working. You should follow the button styling: Active, Hover, and Pressed. Please try to match this button according to the Figma design asked by the previous reviewer.